### PR TITLE
db845c update kernel and qrtr/rmtfs upgrades

### PR DESCRIPTION
**Pull request by alimon**

### CODE REVIEW

File: recipes-support/rmtfs/rmtfs_git.bb [hunk 1]
```diff
@@ -18,6 +18,11 @@ S = "${WORKDIR}/git"
 
 EXTRA_OEMAKE = "'LDFLAGS=${TARGET_LDFLAGS} -L${STAGING_LIBDIR} -lqrtr -ludev'"
 
+do_install_prepend_dragonboard-845c () {
```
Comment (ndechesne) [inline recipes-support/rmtfs/rmtfs_git.bb]: why is that needed? what happens if we don't have this patch? 
we added that in b03212707cec5c49bde4e1d6afb58513e1731318, and I would like to understand how to deal with 8016 and 845. If we need different files for each platform, we probably should use apq8016/rmtfs.service instead of patching them in the recipe.

Comment (alimon) [reply recipes-support/rmtfs/rmtfs_git.bb]: The kernel config that I compiled the module as part of the kernel, I can change it to compile as a module and yes will be different _pas intead of _pil.

Comment (ndechesne) [reply recipes-support/rmtfs/rmtfs_git.bb]: everything that can be a module should be a module in general. so if you can make it a module then it's better. 
looks like there is a newer version of rmtfs that we should use that would automatically start the modem. however i don't think we've started the modem yet, only adsp for audio.

Comment (alimon) [reply recipes-support/rmtfs/rmtfs_git.bb]: @ndechesne Ok changed to use modules and don't enable rmtfs yet.

File: recipes-multimedia/alsa/alsa-lib_1.1.%.bbappend [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${BPN}:"
+
+SRC_URI += " \
```
Comment (ndechesne) [inline recipes-multimedia/alsa/alsa-lib_1.1.%.bbappend]: we should not modify SRC_URI without an override. and that change makes alsa-lib package MACHINE_ARCH.

Comment (alimon) [reply recipes-multimedia/alsa/alsa-lib_1.1.%.bbappend]: @ndechesne Just update the patch to only apply when is sdm845 or apq8096.

**Comment by ndechesne** [comment]
pushed to master as well
